src/nim_status_client.nim

```diff
@@ -19,6 +19,8 @@ when defined(windows):
 logScope:
   topics = "status-app"
 
+const fleetJson = staticRead("../fleets.json")
```

Comment:
Hmm, we had a different path for Windows+production below...

Reply:
That's the runtime path. But it's not needed anymore since we're bundling the json at compile time.

Reply:
I presume that so far the `fleets.json` was bundled into the final package, what's no longer needed. Shouldn't be some deployment scripts updated to exclude that file?

Reply:
> I presume that so far the `fleets.json` was bundled into the final package, what's no longer needed. Shouldn't be some deployment scripts updated to exclude that file?

Updated. Started a deeper investigation on the `fleets.json` file to find out when should we update it (if so). But for now I'm just amending here the build scripts not to include it in the deployment flow. I'll add another follow-up task to conclude when the `fleets.json` file should be updated.